docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -0,0 +1,146 @@
+[[synthetic-run-tests]]
+= Run synthetic tests
+
+++++
+<titleabbrev>Run synthetic tests</titleabbrev>
+++++
+
+There are two ways to run synthetic tests:
+
+* If you want to create a single journey and manage it individually, <<synthetics-inline-journey,use an inline monitor>>.
+Inline monitors can be more efficient, but have some limitations like not supporting dependencies or more than one journey.
+* If you have more than one journey, rely on dependencies, or your tests need to live with your
+application code, <<synthetics-test-suite,use a test suite>>.
+
+[discrete]
+[[synthetics-inline-journey]]
+== Use an inline monitor
```

Comment:
Complementing my previous response to @andrewvc's comment: I think here would be a great way to explain what an inline monitor is.